misc/distributed_search/first_config.yaml

```diff
@@ -0,0 +1,69 @@
+name: PeptideCentric.v1
```

Comment:
I feel like there is the risk this config get's out of sync with the master in in
https://github.com/MannLabs/alphadia/blob/main/alphadia/constants/default.yaml

Do we need a copy?

Reply:
I also see the problems with the duplicated configs..
also, first and second config differ marginally:
`inference_strategy: heuristic`  vs  `inference_strategy: library`    
`initial_num_candidates: 2` vs  `initial_num_candidates: 5`

if these differences are decisive for the searches, I would maybe have only those two lines in the configs given here..

otherwise/alternatively add two empty configs with some comments on the strategy how to configure first & second search